README.md

```diff
@@ -104,51 +104,51 @@ determining that location is as follows:
-| billing\_account | The ID of the billing account to associate this project with | string | n/a | yes |
-| bucket\_location | The location for a GCS bucket to create (optional) | string | `"US"` | no |
-| bucket\_name | A name for a GCS bucket to create (in the bucket_project project), useful for Terraform state (optional) | string | `""` | no |
-| bucket\_project | A project to create a GCS bucket (bucket_name) in, useful for Terraform state (optional) | string | `""` | no |
-| credentials\_path | Path to a service account credentials file with rights to run the Project Factory. If this file is absent Terraform will fall back to Application Default Credentials. | string | `""` | no |
-| default\_service\_account | Project default service account setting: (delete | depriviledge | keep) | string | `"delete"` | no |
-| disable\_dependent\_services | Whether services that are enabled and which depend on this service should also be disabled when this service is destroyed. | string | `"true"` | no |
-| disable\_services\_on\_destroy | Whether project services will be disabled when the resources are destroyed | string | `"true"` | no |
-| domain | The domain name (optional). | string | `""` | no |
-| folder\_id | The ID of a folder to host this project | string | `""` | no |
-| group\_name | A group to control the project by being assigned group_role (defaults to project editor) | string | `""` | no |
-| group\_role | The role to give the controlling group (group_name) over the project (defaults to project editor) | string | `"roles/editor"` | no |
-| labels | Map of labels for project | map | `<map>` | no |
-| lien | Add a lien on the project to prevent accidental deletion | string | `"false"` | no |
-| name | The name for the project | string | n/a | yes |
-| org\_id | The organization ID. | string | n/a | yes |
-| project\_id | If provided, the project uses the given project ID. Mutually exclusive with random_project_id being true. | string | `""` | no |
-| random\_project\_id | Enables project random id generation. Mutually exclusive with project_id being non-empty. | string | `"false"` | no |
-| sa\_role | A role to give the default Service Account for the project (defaults to none) | string | `""` | no |
-| shared\_vpc | The ID of the host project which hosts the shared VPC | string | `""` | no |
-| shared\_vpc\_subnets | List of subnets fully qualified subnet IDs (ie. projects/$project_id/regions/$region/subnetworks/$subnet_id) | list | `<list>` | no |
-| usage\_bucket\_name | Name of a GCS bucket to store GCE usage reports in (optional) | string | `""` | no |
-| usage\_bucket\_prefix | Prefix in the GCS bucket to store GCE usage reports in (optional) | string | `""` | no |
+| Name                           | Description                                                                                                                                                            |     Type     |     Default      | Required |
```

Comment:
Can you please undo this formatting? The Inputs and Outputs tables should remain as generated by `terraform-docs`.

Reply:
Sure, with next commit

Reply:
@aaron-lane, just fixed this one in [e41724f](https://github.com/terraform-google-modules/terraform-google-project-factory/pull/230/commits/e41724ff27932ad84dc65faf8e6c4d3307beff5e)